rules/data/embedded/embedded-svg.yara

```diff
@@ -0,0 +1,44 @@
+rule svg: medium {
+  meta:
+    description = "Contains SVG file(s)"
```

Comment:
It'd be more accurate if this said: `contains SVG (Scalable Vector Graphics) content`

Reply:
Updated in [`00ddb63` (#704)](https://github.com/chainguard-dev/malcontent/pull/704/commits/00ddb6315da42f1d23b32da5a91293492291a916).